tests/org.jnario.lib.tests/src/org/jnario/lib/tests/ShouldTest.xtend

```diff
@@ -0,0 +1,1035 @@
+/*******************************************************************************
+ * Copyright (c) 2014 BMW Car IT and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.jnario.lib.tests
+
+import org.hamcrest.Matcher
+import org.junit.Test
+
+import static extension org.jnario.lib.Assert.*
+import static org.jnario.lib.Should.*
+
+/**
+ * @author Stéphane Galland - Initial contribution and API
+ */
+public class ShouldTest {
+
+	@Test
+	def __0() {
+		_().assertNull
+	}
+
+	@Test
+	def nullValue_0() {
+		val matcher = nullValue()
+		matcher.matches("abc").assertFalse
+		matcher.matches(1).assertFalse
+		matcher.matches(null).assertTrue
+	}
+
+	@Test
+	def matches_StringFunction1_0() {
+		val matcher1 = matches("abc", [ true ])
+		matcher1.matches("abc").assertTrue
+		matcher1.matches(1).assertTrue
+		val matcher2 = matches("abc", [ false ])
+		matcher2.matches("abc").assertFalse
+		matcher2.matches(1).assertFalse
+	}
+
+	@Test
+	def matches_StringFunction1_1() {
+		val matcher1 = matches("abc", [ true ])
+		matcher1.matches(null).assertTrue
+		val matcher2 = matches("abc", [ false ])
+		matcher2.matches(null).assertFalse
+	}
+
+	@Test
+	def should_be_TFunction1_0() {
+		val obj1 = new Object
+		val obj2 = new Object
+		should_be( obj1, [ 
+			assertSame(obj1, it)
+			true
+		] ).assertTrue
+		should_be( obj2, [ 
+			assertSame(obj2, it)
+			false
+		] ).assertFalse
+	}
+	
+	@Test
+	def should_be_TFunction1_1() {
+		should_be( null, [ 
+			assertNull(it)
+			true
+		] ).assertTrue
+		should_be( null, [ 
+			assertNull(it)
+			false
+		] ).assertFalse
+	}
+
+	@Test
+	def should_be_TT_0() {
+		val obj1 = "a"
+		val obj2 = "b"
+		val obj3 = "a"
+		should_be(obj1, obj1).assertTrue
+		should_be(obj1, obj2).assertFalse
+		should_be(obj1, obj3).assertTrue
+	}
+
+	@Test
+	def should_be_TT_1() {
+		val obj1 = #[ "a" ]

```

Comment:
The use of ``#[]`` with spaces throughout the whole PR is inconsistent with the style [already](https://github.com/sebastianbenz/Jnario/blob/master/plugins/org.jnario.standalone/src/org/jnario/compiler/JnarioStandaloneCompiler.xtend#L59) [established](https://github.com/sebastianbenz/Jnario/blob/master/plugins/org.jnario.feature/src/org/jnario/feature/jvmmodel/StepsProvider.xtend#L20).